runtime/parachains/src/shared.rs

```diff
@@ -38,6 +39,77 @@ pub(crate) const SESSION_DELAY: SessionIndex = 2;
 #[cfg(test)]
 mod tests;
 
+/// The maximum amount of relay-parent lookback.
+// TODO [now]: put this in the configuration module (https://github.com/paritytech/polkadot/issues/4841).
```

Comment:
```suggestion
// TODO: put this in the configuration module (https://github.com/paritytech/polkadot/issues/4841).
```

I use TODO [now] for stuff that should be done in the current PR. If there's an open issue, then the [now] should be removed.